src/pal/src/sync/cs.cpp

```diff
@@ -627,7 +627,7 @@ namespace CorUnix
         pPalCriticalSection->LockCount         = 0;
         pPalCriticalSection->RecursionCount    = 0;
         pPalCriticalSection->SpinCount         = dwSpinCount;
-        pPalCriticalSection->OwningThread      = NULL;
+        pPalCriticalSection->OwningThread      = (SIZE_T)NULL;
```

Comment:
`pPalCriticalSection->OwningThread      = 0;` should work too, with less ceremony.